client/gutenberg/extensions/subscriptions/editor.js

```diff
@@ -0,0 +1,60 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+//import './editor.scss';
+import edit from './edit';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import JetpackBlockType from 'gutenberg/extensions/presets/jetpack/utils/jetpack-block-type';
+
+const SubscriptionBlock = new JetpackBlockType( 'subscriptions', {
+	title: __( 'Subscriptions' ),
+
+	description: (
+		<Fragment>
+			<p>{ __( 'Invite visitors to subscribe to your site.' ) }</p>
+		</Fragment>
+	),
+
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 208 128">
```

Comment:
I believe this is the Markdown SVG, so this would need to be updated:

<img width="467" alt="screenshot 2018-11-26 at 15 05 26" src="https://user-images.githubusercontent.com/426388/49018930-f866c400-f18c-11e8-9130-fd6a29017c82.png">